src/topkg_build.ml

```diff
@@ -33,11 +38,16 @@ let with_dir b dir = { b with dir }
 let nop = fun _ -> Ok ()
 
 let cmd c os files =
-  Topkg_os.Cmd.run @@ Topkg_cmd.(build_cmd c os %% of_list files)
+  Topkg_os.Cmd.run @@
+    Topkg_cmd.(build_cmd ~use_toolchain:false c os %% v "-just-plugin") >>= fun () ->
+  Topkg_os.Cmd.run @@
+    Topkg_cmd.(build_cmd ~use_toolchain:true c os %% of_list files)
```

Comment:
Your questions are completely obscure. In which context ? Configuration keys can be looked up with the configuration that is given to you in some contexts (e.g. package description [`describe`](http://erratique.ch/software/topkg/doc/Topkg.Pkg.html#VALdescribe), build description [`build`](http://erratique.ch/software/topkg/doc/Topkg.Pkg.html#VALbuild)). See the opaque [API documentation](http://erratique.ch/software/topkg/doc/Topkg.Conf.html).